crates/re_types/src/archetypes/asset3d_ext.rs

```diff
@@ -0,0 +1,39 @@
+use crate::components::MediaType;
+
+use super::Asset3D;
+
+impl Asset3D {
+    /// Creates a new [`Asset3D`] from the file contents at `path`.
+    ///
+    /// The [`MediaType`] will first be guessed from the file extension, then from the file
+    /// contents if needed.
+    ///
+    /// If no [`MediaType`] can be guessed at the moment, the Rerun Viewer will try to guess one
+    /// from the data at render-time. If it can't, rendering will fail with an error.
```

Comment:
What is the motivation for pushing guess-from-bytes to render-time, instead of doing it in this function?

Reply:
If we guess both from path and bytes right here in this function, we could even warn if the guesses don't agree!

Reply:
I see now - then we need to guess it from bytes on the SDK side, which means doing so in all languages.